### ISSUE TITLE
ResultSetMetadata from a `select * from table` statement will indicate that primary key columns are not writable

### ISSUE THREAD

**Issue by jpoernomoatsafe** [olavloite/spanner-jdbc#77]
Hello, 

Suppose we have a table such as:
```
CREATE TABLE `jpoetest` (`mytext` string(200), `myint` int64) PRIMARY KEY (`myint`)
```

and we're reading such table:
```
Statement statement = getConnection().createStatement();
ResultSet resultSet = statement.executeQuery("SELECT * from `jpoetest` where 1=0");
ResultSetMetaData metaData = resultSet.getMetaData();
if (metaData.isWritable(1)) {
    // this is as expected, "mytext" is writable
}
if (!metaData.isWritable(2)) {
    // why is myint reported as not writable? 
}
```

Currently, `metaData.isWritable(2)` would return false, though the column is actually writable. 
We were expecting that it would return true. 

version: 
spanner-jdbc-1.0.8.jar

**Comment by olavloite** [comment]
The reason that this method returns false for column 2 (`myint`), is that it is part of the primary key. Primary key values are not updatable in Cloud Spanner. You can however insert values. So it is writable when you do an insert, but is no longer writable when you do an update.

The JDBC specification is vague when it comes to this method: It does not specify what the method should return when the column is insertable but not updatable. I chose to implement this method based on whether the column is updatable, as it otherwise would always return true. Does this cause problems with your software?

**Comment by olavloite** [comment]
Fixed for version 1.0.9.
